models/event.go

```diff
@@ -23,13 +23,13 @@ import (
 
 // Event represents a single measurable event read from a device
 type Event struct {
-	ID          string    `json:"id" codec:"omitempty"`       // ID uniquely identifies an event, for example a UUID
-	Pushed      int64     `json:"pushed" codec:"omitempty"`   // Pushed is a timestamp indicating when the event was exported. If unexported, the value is zero.
-	Device      string    `json:"device" codec:"omitempty"`   // Device identifies the source of the event, can be a device name or id. Usually the device name.
-	Created     int64     `json:"created" codec:"omitempty"`  // Created is a timestamp indicating when the event was created.
-	Modified    int64     `json:"modified" codec:"omitempty"` // Modified is a timestamp indicating when the event was last modified.
-	Origin      int64     `json:"origin" codec:"omitempty"`   // Origin is a timestamp that can communicate the time of the original reading, prior to event creation
-	Readings    []Reading `json:"readings" codec:"omitempty"` // Readings will contain zero to many entries for the associated readings of a given event.
+	ID          string    `json:"id" codec:"id,omitempty"`             // ID uniquely identifies an event, for example a UUID
+	Pushed      int64     `json:"pushed" codec:"pushed,omitempty"`     // Pushed is a timestamp indicating when the event was exported. If unexported, the value is zero.
+	Device      string    `json:"device" codec:"device,omitempty"`     // Device identifies the source of the event, can be a device name or id. Usually the device name.
+	Created     int64     `json:"created" codec:",created,omitempty"`  // Created is a timestamp indicating when the event was created.
```

Comment:
Is that leading comma in the codec tag supposed to be there?

Reply:
No, that is a mistake. Great catch.